README.md

```diff
@@ -41,21 +40,43 @@ const loggerConfig = {
   },
 
   // configs for log collecting service (fluentD)
-  FluentdLogger: {
+  Fluentd: {
     // override minimum log level (optional)
-    minLogLevel: LOG_LEVEL.INFO,
+    logLevel: Level.INFO,
 
     // log collector URL
     collectorUrl: LOG_COLLECTOR_URL,
   },
 
   // configs for logging to console
-  ConsoleLogger: {
+  Console: {
     // override minimum log level (optional)
-    minLogLevel: LOG_LEVEL.ERROR,
+    logLevel: Level.ERROR,
   },
 };
+```
+
+Then, there are two ways to continue, **the easy way** and **the complete way**:
 
+### Easy Way
+
+This is the simple way and should cover ~90% of the use cases.
+```js
+const logger = require('@17media/node-logger').CreateLogger(config)('some:label');
```

Comment:
Should this be `createLogger` or just `require('@17media/node-logger')(config)` ?